src/test/java/redis/clients/jedis/commands/jedis/ListCommandsTest.java

```diff
@@ -875,4 +876,25 @@ public void blmove() {
     assertByteArrayListEquals(Collections.singletonList(b3), jedis.lrange(bbar, 0, -1));
     assertByteArrayListEquals(Arrays.asList(b1, b2), jedis.lrange(bfoo, 0, -1));
   }
+
+  @Test
+  public void lmpop(){
+    KeyedListValueElements keyedListValueElements;
```

Comment:
@sazzad16 added all test cases here.String args are working fine.Please check the approach